zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -733,6 +789,21 @@ impl<'a, F: Field> EVMConstraintBuilder<'a, F> {
         );
     }
 
+    #[cfg(feature = "dual_bytecode")]
+    pub(crate) fn bytecode2_length(&mut self, code_hash: Expression<F>, value: Expression<F>) {
+        self.add_lookup(
+            "Bytecode2 (length)",
```

Comment:
seems the name `Bytecode1` and `Bytecode2` is mixed used?   what about change all `Bytecode1` to `Bytecode2`?

Reply:
changed to all Bytecode1, original bytecode remains `Bytecode`  to make less change